### DISCUSSION TITLE
Support for PowerShell Core 6.x

### DISCUSSION THREAD

**Discussion by pejuborg** [PowerShell/PowerShell#19449] [Q&A] [answered]
Hi,

Is PowerShell Core 6.x supported?

/Per

**Comment by iSazonov** [comment, marked as answer]
No.
https://learn.microsoft.com/en-us/powershell/scripting/install/powershell-support-lifecycle?view=powershell-7.3#powershell-end-of-support-dates

**Reply by pejuborg** [reply]
Could not ask for a faster and better answer. 👍
